Mathlib/RingTheory/Length.lean

```diff
@@ -126,6 +126,18 @@ lemma Module.length_bot :
     Module.length R (⊤ : Submodule R M) = Module.length R M := by
   rw [Module.length_submodule, Module.length_eq_height]
 
+lemma Submodule.height_lt_top [IsArtinian R M] [IsNoetherian R M] (N : Submodule R M) :
+    Order.height N < ⊤ := by
+  simpa only [← Module.length_submodule] using Module.length_ne_top.lt_top
+
+lemma Submodule.height_strictMono [IsArtinian R M] [IsNoetherian R M] :
+    StrictMono fun N : Submodule R M ↦ Order.height N :=
```

Comment:
```suggestion
    StrictMono (Order.height : Submodule R M → ℕ∞) :=
```
I find this more readable, but do as you prefer.

Reply:
Now that I see it, I think I prefer your spelling too :)